docs/site/todo-list-tutorial-controller.md

```diff
@@ -0,0 +1,169 @@
+---
+lang: en
+title: "Add TodoList and TodoList's Todo Controller"
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-controller.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList and TodoList's Todo Controller
+---
+
+### Controllers with related models
+
+Defining business logic to handle requests to related models isn't too different
+from handling requests to standalone models. We'll create controllers to handle
+requests for todo-lists and todo items under a todo-list.
+
+### Create TodoList controller
+
+Starting with creating a controller for our `TodoList` routes, run the CLI
```

Comment:
I suggest to say `Run the CLI command for creating a RESTful CRUD controller for our 'TodoList' routes with the following inputs:` or start with `Let's run`/`Let's start by running` for the same sentence.